EasyClangComplete.py

```diff
@@ -112,6 +112,8 @@ def on_plugin_loaded(self):
         # re-activated. Force active view initialization in that case.
         self.on_activated_async(sublime.active_window().active_view())
 
+        self.saving = False
```

Comment:
Why do we need this? It seems that it doesn't really belong in `EasyClangComplete` class which should just manage other objects. But I may be wrong on this one.